openslides/core/urls.py

```diff
@@ -7,4 +7,8 @@
     url(r'^core/url_patterns/$',
         views.UrlPatternsView.as_view(),
         name='core_url_patterns'),
+
+    # View for the projectors are handelt by angular.
+    url(r'^projector.*$',
```

Comment:
The base entry point can not be moved into the core.py because it has to be the last entry.

I would like, that only apps are included in the global urls.py and the projector is not an app, but one view of the core app. So I would like to keep it this way.